pkg/manager/member/startscript/v1/template.go

```diff
@@ -97,6 +97,10 @@ fi
 ARGS="${ARGS}  --plugin-dir  {{ .PluginDirectory  }} --plugin-load {{ .PluginList }}  "
 {{- end }}
 
+{{- if .EnableBootstrapSQL }}
+ARGS="${ARGS}  --initialize-sql-file={{ .BootstrapSQLFile }}"
+{{- end }}
+
```

Comment:
This will cause existing TiDB pods to restart when upgrading TiDB-Operation to a version with this PR.

How about injecting this item into `/etc/tidb/tidb.toml` without any extra lines if not specified?

/cc @KanShiori